sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsView.java

```diff
@@ -539,95 +539,96 @@ public HorizontalLayout createStatusFilterBar() {
+						&& UserProvider.getCurrent().hasUserRight(UserRight.EVENTGROUP_LINK)) {
+						bulkActions.add(
+							new MenuBarHelper.MenuBarItem(
+								I18nProperties.getCaption(Captions.actionGroupEvent),
+								VaadinIcons.FILE_TREE,
+								mi -> grid.bulkActionHandler(
+									items -> ControllerProvider.getEventController()
+										.linkAllToGroup(eventGrid.asMultiSelect().getSelectedItems(), eventGrid))));
 					}
-				}
-				if (UserProvider.getCurrent().hasUserRight(UserRight.EVENT_ARCHIVE)) {
-					bulkActions.add(
-						new MenuBarHelper.MenuBarItem(
-							I18nProperties.getCaption(Captions.actionArchiveCoreEntity),
-							VaadinIcons.ARCHIVE,
-							mi -> grid
-								.bulkActionHandler(items -> ControllerProvider.getEventController().archiveAllSelectedItems(items, eventGrid), true),
-							EntityRelevanceStatus.ACTIVE.equals(eventCriteria.getRelevanceStatus())));
 					bulkActions.add(
 						new MenuBarHelper.MenuBarItem(
-							I18nProperties.getCaption(Captions.actionDearchiveCoreEntity),
-							VaadinIcons.ARCHIVE,
+							I18nProperties.getCaption(Captions.ExternalSurveillanceToolGateway_send),
+							VaadinIcons.SHARE,
 							mi -> grid.bulkActionHandler(
 								items -> ControllerProvider.getEventController()
-									.dearchiveAllSelectedItems(eventGrid.asMultiSelect().getSelectedItems(), eventGrid),
-								true),
-							EntityRelevanceStatus.ARCHIVED.equals(eventCriteria.getRelevanceStatus())));
-				}
-				if (UserProvider.getCurrent().hasUserRight(UserRight.EVENTGROUP_CREATE)
-					&& UserProvider.getCurrent().hasUserRight(UserRight.EVENTGROUP_LINK)) {
+									.sendAllSelectedToExternalSurveillanceTool(eventGrid.asMultiSelect().getSelectedItems(), eventGrid)),
+							UserProvider.getCurrent().hasUserRight(UserRight.EXTERNAL_SURVEILLANCE_SHARE)
+								&& FacadeProvider.getExternalSurveillanceToolFacade().isFeatureEnabled()));
+
+					if (isDocGenerationAllowed()) {
+						bulkActions.add(
+							new MenuBarHelper.MenuBarItem(I18nProperties.getCaption(Captions.bulkActionCreatDocuments), VaadinIcons.FILE_TEXT, mi -> {
+								grid.bulkActionHandler(items -> {
+
+									EventGrid eventGrid1 = (EventGrid) this.grid;
+									List<EventReferenceDto> references = eventGrid1.asMultiSelect()
+										.getSelectedItems()
+										.stream()
+										.map(EventIndexDto::toReference)
+										.collect(Collectors.toList());
+									if (references.size() == 0) {
+										new Notification(
+											I18nProperties.getString(Strings.headingNoEventsSelected),
+											I18nProperties.getString(Strings.headingNoEventsSelected),
+											Notification.Type.WARNING_MESSAGE,
+											false).show(Page.getCurrent());
+
+										return;
+									}
+
+									ControllerProvider.getDocGenerationController().showEventDocumentDialog(references);
+								});
+							}));
+					}
+				} else if (UserProvider.getCurrent().hasUserRight(UserRight.EVENT_DELETE)) {
```

Comment:
Should be enough an else here because the other EntityRelevanceStatuses (ACTIVE, ARCHIVED, ACTIVE_AND_ARCHIVED,) will be handled when the status is not deleted.